README.md

```diff
@@ -196,14 +201,16 @@ fn get_acceptor() -> TlsAcceptor {
 }
 
 fn main() {
-    let addr = "localhost:10000".first_socket_addr();
     let mut core = reactor::Core::new().unwrap();
     let acceptor = get_acceptor();
-    HelloServer.listen(addr, server::Options::default()
-                                 .handle(core.handle())
-                                 .tls(acceptor)).wait().unwrap();
-    let options = client::Options::default().handle(core.handle()
-                      .tls(client::tls::Context::new("foobar.com").unwrap()));
+    let addr = HelloServer.listen("localhost:10000".first_socket_addr(),
+                                  server::Options::default()
```

Comment:
I think this should be `server::Options::from(core.handle())...` like the other ones since `Default` is gone.